README.md

```diff
@@ -27,6 +27,11 @@ integration from the Jetpack Benchmark library.
 This sample compares how different ViewGroups
 (ConstraintLayout vs traditional layouts) affect the UI performance.
 
+## [MacrobenchmarkSample](MacrobenchmarkSample)
```

Comment:
Move MacrobenchmarkSample above ConstraintLayout, as that sample's not actively maintained.